openapi/stellar-disbursement-platform/main.yml

```diff
@@ -1831,6 +1831,180 @@ paths:
         - application/json
       security:
         - BearerAuth: []
+  
+  /tenants:
+    get:
+      summary: Get All Tenants
+      responses:
+        "200":
+          description: "This endpoint returns all registered tenants on the system"
+          schema:
+            $ref: "#/definitions/Tenants"
+        "401":
+          $ref: "#/responses/UnauthorizedResponse"
+      operationId: Get All Tenants
+      tags:
+        - Admin
+      parameters: []
+      produces:
+        - application/json
+      security:
+        - BasicAuth: []
+    post:
+      summary: Create Tenant
+      responses:
+        "201":
+          description: "Provision a new tenant"
+          schema:
+            $ref: "#/definitions/Tenant"
+        "401":
+          $ref: "#/responses/UnauthorizedResponse"
+      operationId: Create Tenant
+      tags:
+        - Admin
+      parameters:
+        - name: body
+          in: body
+          schema:
+            type: object
+            properties:
+              name:
+                type: string
+                description: The tenant name. It should match the pattern `^[a-z-]+$`
+              owner_email:
+                type: string
+                description: The owner user e-mail address
+              owner_first_name:
+                type: string
+                description: The owner user first name
+              owner_last_name:
+                type: string
+                description: The owner user last name
+              organization_name:
+                type: string
+                description: The organization name
+              email_sender_type:
+                type: string
+                enum: [ DRY_RUN, AWS_EMAIL ]
+                description: The e-mail sender provider.
+              sms_sender_type:
+                type: string
+                enum: [ DRY_RUN, AWS_SMS, TWILIO_SMS ]
+                description: The SMS sender provider.
+              base_url:
+                type: string
+                description: The SDP backend server's base URL.
+              sdp_ui_base_url:
+                type: string
+                description: The SDP UI/dashboard Base URL.
+              distribution_account:
```

Comment:
I see that `TenantRequest` takes a `distrubution_account` but I don't see that value being used anywhere. Please double check that

Reply:
That's true, I believe we missed adding it to the tenant after the tenant provision.

Reply:
As discussed on the stand-up, the field was removed.